Dockerfile

```diff
@@ -17,6 +17,8 @@ RUN set -ex\
 		memcached \
 		nginx \
 		libpq-devel \
+		libjpeg-turbo \
+		libjpeg-turbo-devel \
```

Comment:
Why do you need the devel package in the base image? Usually they are needed only in the build image.

Reply:
sorry about that, I thought both might be needed so tested with both packages. Just now I tested by removing devel package, which is working fine. Will update accordingly